basename+date+fold+nl+nproc+shuf+uname: use UResult

thomasqueirozb opened the pull request:
Related to #2464

sylvestre commented:
date GNU tests seem to fail:
`Error: GNU test failed: tests/misc/date-tz. tests/misc/date-tz is passing on 'master'. Maybe you have to rebase?`

thomasqueirozb commented:
> date GNU tests seem to fail: `Error: GNU test failed: tests/misc/date-tz. tests/misc/date-tz is passing on 'master'. Maybe you have to rebase?`

Fixed it. Problem was the original implementation just showed an error and returned 0 and I had changed to return an USimpleError with code 1. Reverted this change
